sites/public/lib/hooks.ts

```diff
@@ -34,17 +34,22 @@ export const useFormConductor = (stepName: string) => {
   return context
 }
 
-function filterStringFromFilters(filters: ListingFilterParams) {
-  if (!filters || filters.neighborhood == "") return ""
-
-  // Only `neighborhood` filter is currently supported.
-  return `&filter[$comparison]==&filter[${ListingFilterKeys.neighborhood}]=${filters.neighborhood}`
+function backendFilterParamsFromFilters(filters: ListingFilterParams) {
+  if (!filters) return ""
+  let filterString = ""
+  for (const filterKey in ListingFilterKeys) {
+    const value = filters[filterKey]
+    if (value && value != "") {
+      filterString += `&filter[$comparison]==&filter[${filterKey}]=${value}`
+    }
+  }
+  return filterString
```

Comment:
I think you're already planning this, but this should probably be in a util file since it's not itself a hook

Reply:
I'll move it over to a filter helper file, it's coming up in the next PR. Added a TODO for giggles